Sources/Comet/TokenProviding/TokenProvidingHttpError.swift

```diff
@@ -5,6 +5,11 @@
 import Foundation
 
 public struct TokenProvidingHttpError {
-    let code: Int
-    let data: Data
+    public init(code: Int, data: Data) {
+        self.code = code
+        self.data = data
+    }
+
+    public let code: Int
```

Comment:
Nejsem si jisty, jestli toto potrebujeme jako Public?

Reply:
Ani já ne, ale vzhledem k tomu, že implementace TokenProvider je v dané aplikaci, může být potřeba s touto chybou pracoast.

Reply:
Ale asi to nicemu nevadi. Kdyztak to pak zmenime.